pyaedt/modules/LayerStackup.py

```diff
@@ -12,12 +12,12 @@
 
 @aedt_exception_handler
 def str2bool(str0):
-    """
+    """Convert a string to a Boolean value.
 
     Parameters
     ----------
-    str0 :
-        
+    str0 : text
```

Comment:
```suggestion
    str0 : str
```
It's a bad parameter name and that probably threw you off.